src/module/os.c

```diff
@@ -109,6 +109,20 @@ void processExit(WrenVM* vm) {
   uv_stop(getLoop());
 }
 
+// chdir_(dir)
+void processChdir(WrenVM* vm)
+{
+  wrenEnsureSlots(vm, 1);
+  char* dir = cli_strdup(wrenGetSlotString(vm, 1));
```

Comment:
I was looking at the node.js [implementation](https://github.com/nodejs/node/blob/d14878dc7c6d23038b59e837fcd61f8b35fced30/src/node_process_methods.cc#L81) -- is there a need to validate that the input is a String?

Reply:
`wrenGetSlotString` does that in the debug build... not our problem in production the assumption is that code is properly written.  Might be a good time to ponder if someone using Wren for scripting should be running debug builds though.